islandora_basic_collection.module

```diff
@@ -83,18 +83,17 @@ function islandora_basic_collection_menu() {
 }
 
 /**
- * Implements hook_menu_local_tasks_alter().
- * Reorders the collection management links to move 'Add an Object' to the top using 'weight'.
+ * Implements hook_menu_local_tasks_alter(). Move 'Add an Object' link to the top using 'weight'.
```

Comment:
it's this line breaking DCS, it must be <=80 characters.

Reply:
Got it, thanks.

Reply:
Travis still unhappy:

` +86: [minor] Format should be "* Implements hook_foo().", "* Implements hook_foo_BAR_ID_bar() for xyz_bar().",, "* Implements hook_foo_BAR_ID_bar() for xyz-bar.html.twig.", or "* Implements hook_foo_BAR_ID_bar() for xyz-bar.tpl.php.".`

So I shouldn't have more than one line of comments in there, AND I shouldn't put the explanation in the same line as the hook implementation note? What is the correct approach?

Reply:
As Diego said. I use the rule when in doubt leave a line.
```
/**
 * Implements hook_menu_local_tasks_alter().
 * 
 * Move 'Add an Object' link to the top using 'weight'.
 */
```